packages/node_modules/@webex/internal-plugin-avatar/src/avatar.js

```diff
@@ -107,24 +108,17 @@ const Avatar = WebexPlugin.extend({
       }))
       .then((file) => this.upload({
         api: 'avatar',
-        resource: 'profile',
+        resource: `profile/${this.webex.internal.device.userId}/session`,
         file: file[0],
         phases: {
           upload: {
-            $uri: function $uri(session) {
-              return session.url;
-            }
+            $uri: (session) => session.url
           },
           finalize: {
             method: 'PUT',
             api: 'avatar',
-            // eslint-disable-next-line no-unused-vars
-            $resource: function $resource(session) {
-              return `profile/${session.id}`;
-            },
-            $body: function $body(session) {
-              return session;
-            }
+            $resource: (session) => `profile/${this.webex.internal.device.userId}/session/${session.id}`,
+            $body: (session) => session
           }
         }
```

Comment:
@aimeex3 Hi, even though the new api in Avatar is also a PUT request, you don't have to send any body. It will not be used. Otherwise look fine. 

tortrett@cisco.com

Reply:
thanks @toretre, i will update and remove the body